charts/keep/README.md

```diff
@@ -1,6 +1,6 @@
 # keep
 
-![Version: 0.0.5](https://img.shields.io/badge/Version-0.0.5-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.2.1](https://img.shields.io/badge/AppVersion-0.2.1-informational?style=flat-square)
+![Version: 0.1.17](https://img.shields.io/badge/Version-0.1.17-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.25.2](https://img.shields.io/badge/AppVersion-0.25.2-informational?style=flat-square)
```

Comment:
is there a way somehow to use the value from `charts.yaml` so we won't need to update two files?

Reply:
I don't think so as it is autogenerated from the helm-docs pre-commit hook and need to be generated again on every version change.